udata_front/theme/gouvfr/assets/js/composables/useDropdown.ts

```diff
@@ -11,7 +11,7 @@ export function useDropdown<T extends OptionWithId>(options: MaybeRefOrGetter<Ar
   const { expanded, handleKeyPressForCollapse, hide, registerBackgroundEvent, removeBackgroundEvent, show, uid } = useCollapse();
   const { focusOut, handleKeyPressForActiveDescendant, isSelected, select, selected, selectedOption, NOT_MOVED_YET, ALREADY_MOVED_DOWN } = useActiveDescendant(options);
 
-  function showAndSelectIfQuery(q: MaybeRefOrGetter<string>) {
+  function showAndSelectIfQuery() {
```

Comment:
q should be here

Reply:
no need to pass q as it's already set earlier in the code